hera_cal/tests/test_omni.py

```diff
@@ -962,6 +962,27 @@ def test_single_file_execution_omni_apply_overwrite(self):
         objective_file = os.path.join(DATA_PATH, 'zen.2457698.40355.xx.HH.uvcAAO')
         if os.path.exists(objective_file):
             shutil.rmtree(objective_file)
+        os.makedirs(objective_file)
+        o = omni.get_optionParser('omni_apply')
+        omni_file = os.path.join(DATA_PATH, 'test_input', xx_ocal)
+        vis_file = os.path.join(DATA_PATH,  xx_vis)
+        cmd = "-p xx --omnipath={0} --extension=O --overwrite {1}".format(omni_file, vis_file)
+
+        opts, files = o.parse_args(cmd.split())
+        omni.omni_apply(files, opts)
+        nt.assert_true(os.path.exists(objective_file))
+	# make sure its a miriad file
+	uvd = UVData()
+	uvd.read_miriad(objective_file)
+	# make sure a metadata column is set correctly
+        nt.assert_equal(uvd.Nants_data, 19)
+        # clean up when we're done
+        shutil.rmtree(objective_file)
+
+    def test_single_file_execution_omni_apply_overwrite(self):
```

Comment:
It looks like the name of this test is the same as the one above it. Are they supposed to be different? Also, it looks to me like some of the columns are out of alignment here, but not in my text editor--maybe it's a tabs vs spaces thing?

Reply:
ah, yeah that was a mistake...